monero-harness/src/image.rs

```diff
@@ -25,7 +25,7 @@ impl Image for Monerod {
     type EntryPoint = str;
 
     fn descriptor(&self) -> String {
-        "melotools/monero:v0.17.2.0".to_owned()
+        "rinocommunity/monero:v0.17.2.0".to_owned()
```

Comment:
This need to be changed to 17.3.0 because there is no tag for 17.2.0 Rinocommunity

Reply:
Yes I'd suggest trying 0.17.3 to see if the CI passes, since a patch version upgrade should not be a breaking change